.github/workflows/run-e2e-tests-saas.yaml

```diff
@@ -0,0 +1,31 @@
+name: Cypress E2E SaaS Tests
+on: push
+jobs:
+  cypress-saas-run:
+    # If you want this job to run on your fork, remove the below "if" line.
+    # You will need to "fix" the tests as they are specific to Adobe Commmerce's
+    # demo backend.
+    if: github.repository == 'hlxsites/aem-boilerplate-commerce'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install root dependencies
+        run: npm ci
+      - name: Install adobe aem cli dependencies
+        run: npm install -g @adobe/aem-cli
+      - name: Start server in the background
+        run: aem up --url https://main--boilerplate-accs--adobe-commerce.aem.live/ &
+      - name: Install Cypress and run tests
+        uses: cypress-io/github-action@v6
+        with:
+          working-directory: cypress
+          browser: chrome
+          wait-on: 'http://localhost:3000'
+          config: baseUrl=http://localhost:3000
+          command: npm run cypress:saas:run
```

Comment:
Have you seen this? It looks like using `command` removes need for `browser` or `config` - but you still define them.

<img width="866" alt="image" src="https://github.com/user-attachments/assets/1203f8d8-c5fd-4349-bbf6-1731bcf98197" />

And what is interesting is the `cypress:run` and `cypress:saas:run` commands do not include `browser` or `config` flags.

If these flags are required (browser, config) then we should add to the `run` commands. Either way, we should delete them from the yaml here.

Reply:
We will see what happens with https://github.com/hlxsites/aem-boilerplate-commerce/pull/445/commits/3f3f798d520c41757268bba760fe51fa85c48c05 :)